### DISCUSSION TITLE
如何使用GPU去运行这个代码而不是cpu？

### DISCUSSION THREAD

**Discussion by xiaochang-c** [openai/whisper#583] [Q&A]
我不知道如何在这个项目上使用GPU去运行它，翻看了前面的讨论。尝试将模型加载改为
model = whisper.load_model("small", device="cuda")
发现仍然无法运行，提示我
RuntimeError: Attempting to deserialize object on a CUDA device but torch.cuda.is_available() is False.
但我的设备是有GPU的，tensorflow可以正常调用GPU。

**Comment by xiaochang-c** [comment]
唔，查看了下版本。
import torch
print(torch.cuda.is_available())
print(torch.__version__)
发现默认给我下的是cpu版本。
False
1.13.0+cpu
这就很尴尬了

**Comment by berryyan** [comment]
最后如何解决的呢？

**Reply by berryyan** [reply]
我解决了  思路提供给大家，先确认cuda版本，安装CUDA Toolkit，安装cuDNN（可选），最后重新安装torch；

重复检查版本
import torch
print(torch.cuda.is_available())

看到True 就放心了，然后可以尝试重新运行whisper就等着起飞吧

**Reply by berryyan** [reply]
补充 全称是Pytorch  方便大家自己搜索
